integration-test/royalty.test.js

```diff
@@ -0,0 +1,84 @@
+import { config as loadEnv } from 'dotenv';
+import ganache from 'ganache';
+import { BigNumber, utils } from 'ethers';
+import Auth from '../src/lib/Auth/Auth';
+import SDK from '../src/lib/SDK/sdk';
+import { TEMPLATES } from '../src/lib/NFT/constants';
+
+loadEnv();
+let sdk;
+let account;
+let server;
+let contractObject;
+let publicAddress;
+let owner;
+let thirdUser;
+let privateKeyPublicAddress;
+
+describe('E2E Test: Basic NFT (mint)', () => {
+  jest.setTimeout(120 * 1000);
+
+  beforeAll(async () => {
+    const options = {
+      wallet: {
+        accountKeysPath: 'integration-test/keys.json',
+      },
+      logging: {
+        quiet: true,
+      },
+    };
+
+    server = ganache.server(options);
+    await server.listen(8545);
```

Comment:
All the ganache setup can be extracted into a separate file for easy re-use in all tests (DRY)
- options / server
- parsing keys.json & extracting addr & private keys
- server closing